/****************************************************************************
//	Usagi Engine, Copyright Â© Vitei, Inc. 2013
****************************************************************************/
syntax = "proto2";
import 'nanopb.proto';

package usg.Processor;

message Merge
{
	optional string entityWithID = 1 [(nanopb).max_size = 64];
}
